app/views/embed/_analytics.html.erb

```diff
@@ -0,0 +1,12 @@
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-80J719XWE5"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-80J719XWE5', {
+    'debug_mode': Boolean('<%= Settings.analytics_debug %>'),
+    // 'collection' is a custom event parameter for getting an item's parent collection, if applicable
```

Comment:
How about putting this in a erb comment (e.g. `<%#  like this %>`) so that we don't print it out to the HTML source?